launchpod/src/main/java/com/google/launchpod/data/Item.java

```diff
@@ -0,0 +1,27 @@
+package com.google.launchpod.data;
+
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+
+@JacksonXmlRootElement(localName = "item")
```

Comment:
try getting rid of this

Reply:
getting an error   
 com.fasterxml.jackson.databind.exc.InvalidDefinitionException: No serializer found for class com.google.sps.data.Channel and no properties discovered to create BeanSerializer